src/app.js

```diff
@@ -1,5 +1,167 @@
 const express = require('express');
+const {
+  sayHello,
+  uppercase,
+  lowercase,
+  firstCharacter,
+  firstCharacters,
+} = require('./lib/strings');
+
+const { add, subtract, multiply, divide, remainder } = require('./lib/numbers');
+
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./lib/arrays');
 
 const app = express();
 
+app.use(express.json());
+
+app.get('/strings/hello/:string', (req, res) => {
+  res.status(200).json({ result: sayHello(req.params.string) });
+});
+
+app.get('/strings/upper/:string', (req, res) => {
+  res.status(200).json({ result: uppercase(req.params.string) });
+});
+
+app.get('/strings/lower/:string', (req, res) => {
+  res.status(200).json({ result: lowercase(req.params.string) });
+});
+
+app.get('/strings/first-characters/:string', (req, res) => {
+  const length = req.query.length;
+  if (length) {
+    res.status(200).json({ result: firstCharacters(req.params.string, length) });
+  } else {
+    res.status(200).json({ result: firstCharacter(req.params.string) });
+  }
+});
+
+app.get('/numbers/add/:a/and/:b', (req, res) => {
+  const a = parseInt(req.params.a);
+  const b = parseInt(req.params.b);
+
+  if (Number.isNaN(a) || Number.isNaN(b)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  } else {
+    res.status(200).json({ result: add(a, b) });
+  }
+});
+
+app.get('/numbers/subtract/:a/from/:b', (req, res) => {
+  const a = parseInt(req.params.a);
+  const b = parseInt(req.params.b);
+
+  if (Number.isNaN(a) || Number.isNaN(b)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  } else {
+    res.status(200).json({ result: subtract(b, a) });
+  }
+});
+
+app.post('/numbers/multiply', (req, res) => {
+  const a = req.body.a;
+  const b = req.body.b;
+
+  if (!a || !b) {
```

Comment:
Just as above, the logic works, only some formatting suggestions, for better readability.
Could have two if statements, and get rid of the if else and the else:
````
  if (!a || !b) {
    res.status(400).json({ error: 'Parameters "a" and "b" are required.' });
  }
  if (Number.isNaN(Number(a)) || Number.isNaN(Number(b))) {
    res.status(400).json({ error: 'Parameters "a" and "b" must be valid numbers.' });
  }
  res.status(200).json({ result: multiply(a, b) });
`````